src/lj_ir.h

```diff
@@ -529,6 +529,11 @@ typedef uint32_t TRef;
 */
 
 typedef union IRIns {
+#if LJ_GC64
+#error not yet implemented
+  struct {
+    uint32_t padding;   /* Ensures that gcr/ptr overlap with the proper fields */
```

Comment:
No; making `IRIns` larger than 8 bytes is a bad idea. Using a pair of `IRIns` for 64-bit constants is a better idea.